src/components/CardStats.jsx

```diff
@@ -15,7 +17,9 @@ const CardStats = ({
   milestonesCount,
   type,
   status,
-}) => (
+}) => {
+  BigNumber.config({ DECIMAL_PLACES: config.decimals });
```

Comment:
can we set this globally in `Application.jsx`? instead of each time we import BigNumber?

Reply:
Yeah I did this to start off with but it actually doesn't work. I don't know why.

Reply:
Nevermind. I forgot to delete another config statement which has 18 decimals